oak/server/oak_node.cc

```diff
@@ -138,8 +140,17 @@ static wabt::Result ReadModule(const std::string module_bytes, wabt::interp::Env
   return result;
 }
 
+const std::string Sha256Hash(const std::string& data) {
```

Comment:
It's unusual to have a const return value, so a comment would be good if this is intentionally to prevent use as an lvalue.

Reply:
Actually not intentional.